driver-core/src/main/com/mongodb/internal/operation/CommandOperationHelper.java

```diff
@@ -603,8 +607,8 @@ static void logRetryExecute(final RetryState retryState) {
             Throwable exception = retryState.exception().orElseThrow(Assertions::fail);
             int oneBasedAttempt = retryState.attempt() + 1;
             LOGGER.debug(commandDescription == null
-                    ? format("Retrying an operation due to the error \"%s\"; attempt #%d", exception, oneBasedAttempt)
-                    : format("Retrying the operation %s due to the error \"%s\"; attempt #%d",
+                    ? format("Retrying the operation due to the error \"%s\"; attempt #%d", exception, oneBasedAttempt)
+                    : format("Retrying the operation '%s' due to the error \"%s\"; attempt #%d",
```

Comment:
I doubt _"Retrying **the** operation ..."_ is grammatically correct in a situation when we don't specify what operation is being retried, but having the same longer prefix for two messages seems better.